src/sentry/static/sentry/app/components/userMisery.tsx

```diff
@@ -18,7 +18,12 @@ function UserMisery(props: Props) {
   const {bars, barHeight, miserableUsers, miseryLimit, totalUsers} = props;
 
   const palette = new Array(bars).fill(theme.purpleDarkest);
-  const score = Math.floor((miserableUsers / Math.max(totalUsers, 1)) * palette.length);
+  const rawScore = Math.floor(
+    (miserableUsers / Math.max(totalUsers, 1)) * palette.length
+  );
+
+  const adjustedScore = rawScore > 0 ? rawScore : miserableUsers > 0 ? 1 : 0;
```

Comment:
Would just doing `Math.ceil` give a more accurate visual? Here 0 shows no bars, but all other values below 20% shows 1 bar. Then it increases at 10% increments. And only way to render full bars is if every user is miserable. But using `Math.ceil` the top 10% shows max bars. Also, its just a simpler change

Reply:
Hmm, `ceil` causes the same issue on the top end (like you mentioned). The implication with empty or full bars is that the % is either 0% or 100%, respectively. Either case is likely not ideal.

Near empty appearing as empty and near full appearing as full are both confusing because there is data the user might want to be aware of. eg. 9.9% of users should still be indicated as it's vastly different in certain use cases than 0%, likewise 90.1% is different than 100%.

By 'hinting' with a single bar (or lack thereof in the near-full scenario) we are helping inform the user that there is additional data to look at, as the bars, AFAIK, are meant as a visual aid to the % of users having a problem at a glance, with additional accurate data available on hover.